docs/versioned_docs/version-zenith/reference/979596-cli.mdx

```diff
@@ -10,479 +10,544 @@ import PartialExperimentalDocs from '../partials/_experimental.mdx';
 
 <PartialExperimentalDocs />
 
+## dagger
```

Comment:
Could you also update the page title to be "CLI Reference"? Unrelated to the generation - I just noticed the sidebar link uses that title.

Reply:
This is the `dagger` top level command. If you run it, it'll present the help with available commands.

Reply:
Did you mean the one further up with `# Reference`?

Reply:
I changed the title from "Reference" to "CLI Reference" to match the sidebar. 🙏

Reply:
Thank you!